Dockerfile.build

```diff
@@ -1,9 +1,12 @@
-FROM golang:1.14-buster
+FROM fedora:latest
```

Comment:
Could you fully specify this to grab from Fedora rather then get it at Docker.io, should help rate limits.

Reply:
Done